x-pack/plugins/ingest_manager/public/applications/ingest_manager/components/context_menu_actions.tsx

```diff
@@ -31,10 +33,27 @@ type Props = {
     }
 );
 
-export const ContextMenuActions = React.memo<Props>(({ button, ...props }) => {
-  const [isOpen, setIsOpen] = useState(false);
-  const handleCloseMenu = useCallback(() => setIsOpen(false), [setIsOpen]);
-  const handleToggleMenu = useCallback(() => setIsOpen(!isOpen), [isOpen]);
+export const ContextMenuActions: React.FunctionComponent<Props> = ({
```

Comment:
memoization of this component was removed, is this intentional?

Reply:
Good catch, no I am going to put it back